docs/platform/deployservices.md

```diff
@@ -1,13 +1,37 @@
 # Deploy your platform using SmartSDK recipes
 
+This section will give you pointers to deploy core services of SmartSDK using
+docker recipes, which are mantained in [this repository](https://github.com/smartsdk/smartsdk-recipes)
+ and whose documentation can be read [here](https://smartsdk.github.io/smartsdk-recipes/)
+
+If you haven't already, please take some time to read those docs so that you
+make sure you have properly setup the environment (the docker swarm cluster)
+where the deployment will be done. This is covered in [this section](deployservices.md)
+of this guide.
+
 ## Deploy an Highly Available Context Broker
 
-- TODO
+To deploy Orion Context Broken in HA, please refer to [this section](https://smartsdk.github.io/smartsdk-recipes/data-management/context-broker/ha/readme/) of the
+SmartSDK recipes documentation.
+
+As with the rest of the recipes, the files mentioned in the documentation are
+those located in the same directory where the documentation file is. For this
+case, we are talking about [this directory](https://github.com/smartsdk/smartsdk-recipes/tree/master/recipes/data-management/context-broker/ha).
 
 ## Deploy an Highly Available QuantumLeap
 
-- TODO
+To deploy Orion Context Broken in HA, please refer to [this section](https://smartsdk.github.io/smartsdk-recipes/data-management/quantumleap/readme/) of the
+SmartSDK recipes documentation.
+
+As with the rest of the recipes, the files mentioned in the documentation are
+those located in the same directory where the documentation file is. For this
+case, we are talking about [this directory](https://github.com/smartsdk/smartsdk-recipes/tree/master/recipes/data-management/quantumleap).
 
 ## Deploy an Highly Available IoT Agent
 
-- TODO
+To deploy IoT Agents in HA, please refer to [this section](https://smartsdk.github.io/smartsdk-recipes/iot-services/readme/) of the
```

Comment:
To deploy the a iot-ul agent... (for other agents please refer to ... ) ...

```cd ....
nano env variable
sh deployxxx.sh
```
wait few seconds and test:

1. create a service in the agent
1. create device
1. create a subscpritpion in ctx
1. send data to agent
1. query data from ql
For fine grained configuration refer to...